dags/pull_wys.py

```diff
@@ -79,26 +67,56 @@ def custom_fail_slack_alert(context: dict) -> str:
                  'on_failure_callback': task_fail_slack_alert
                 }
 
-dag = DAG(dag_id = dag_name, default_args=default_args, schedule_interval='0 15 * * *')
-# Run at 3 PM local time every day
-
-with wys_postgres.get_conn() as con:
-    t1 = PythonOperator(
-            task_id = 'pull_wys',
-            python_callable = api_main, 
-            dag = dag,
-            op_kwargs = {'conn':con, 
-                        'start_date':'{{ ds }}', 
-                        'end_date':'{{ ds }}', 
-                        'api_key':api_key}
-                        )
+@dag(dag_id = dag_name,
+     default_args=default_args,
+     schedule_interval='0 15 * * *' # Run at 3 PM local time every day
+     )
+def pull_wys_dag():
+
+    @task
+    def pull_wys(**kwargs):
+        #to connect to pgadmin bot
+        wys_postgres = PostgresHook("wys_bot")
+        
+        #api connection
+        api_conn = BaseHook.get_connection('wys_api_key')
+        api_key = api_conn.password
+
+        with wys_postgres.get_conn() as conn:
+            api_main(start_date = kwargs['ds'],
+                     end_date = kwargs['ds'],
```

Comment:
I think this can be simplified by replacing the `**kwargs` argument with `(ds=None)` in the function definition and then using the `ds` variable directly. 

No need to make the modification. I've made it and will commit and push after it runs as scheduled at 15:00.